### DISCUSSION TITLE
Does having the multiple job groups trigger groups affect quartz performance?

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2272] [Q&A]
I’m running 3000 recurring jobs every minute for an hour. It works fine without any misfires.

and there is the same job group and same trigger group.

but when I run total 3000 jobs every minute, but for two different groups. It restarts misfiring sometimes.

so does having multiple groups affects performance?

In production there could be such 1000 groups.

What is the best way to handle this?

FYI: my service is currently running in single instance and I’m using sql DB for jobs persistence.

**Comment by viveknuna** [comment]
@lahma do you have any input on this?

**Comment by lahma** [comment]
Groups should have much effect in triggering performance as Quartz searches for work based on priority and next fire time.

**Reply by viveknuna** [reply]
@lahma Sorry didn't understand completely. Could you please elaborate a bit?

**Reply by lahma** [reply]
I don't see how group name would affect performance when Quartz is just triggering jobs.
